CONTRIBUTING.md

```diff
@@ -167,6 +167,40 @@ If you are not familiar with creating a Pull Request, here are some guides:
 - http://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request
 - https://help.github.com/articles/creating-a-pull-request/
 
+#### Sync up with the upstream
+
+First, make sure you have set [upstream](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork) by running:
+
+```bash
+git remote add upstream https://github.com/pytorch/ignite
+```
+
+Then you can see if you have set up multiple remote correctly by running `git remote -v`:
+
+```bash
+origin  https://github.com/{YOUR_USERNAME}/ignite.git (fetch)
+origin  https://github.com/{YOUR_USERNAME}/ignite.git (push)
+upstream        https://github.com/pytorch/ignite (fetch)
+upstream        https://github.com/pytorch/ignite (push)
+```
+
+Now you can get the latest development into your forked repository with this:
+
+```bash
+git fetch upstream
+git checkout master
+git merge upstream/master
+```
+
+Rebasing for a PR
```

Comment:
We do not require rebasing on master, actually. As all commit history in the PR is squashed into a single commit. Maybe, we can simply mention that rebasing or merging, everything is possible as we finally squash.

Reply:
I was writing that initially for conflicting PR
I know we squashed and merged into master.
But not sure how you want to suggest for a conflicting PR?

Reply:
> But not sure how you want to suggest for a conflicting PR?

Good question. I'd say that we can suggest to use "merge" and resolve the conflict once.

Reply:
Alright!

Reply:
Maybe we can remove the paragraph about rebasing, right ?

Reply:
Yes if you wish

Reply:
Done!